mobile-app/App.tsx

```diff
@@ -20,21 +25,51 @@ const gamestate: GameState = {
   ],
 }
 
+const Stack = createNativeStackNavigator();
+
 export default function App() {
   return (
-    <View style={styles.container}>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Home">
+        <Stack.Screen name="Home" component={HomeScreen} />
+        <Stack.Screen name="Local" component={LocalScreen} />
+      </Stack.Navigator>
+    </NavigationContainer>
+  );
+}
+
+function HomeScreen(props: { navigation: any }) {
```

Comment:
you should extract it to another file. Also, I'm sure you can give a better type to `navigation`